dle/users/forms.py

```diff
@@ -11,3 +11,8 @@ class MyLabelForm(forms.Form):
     generic_name = forms.CharField(label="Generic name", max_length=255, required=True)
     product_number = forms.CharField(label="Product Number", max_length=255, required=True)
     marketer = forms.CharField(label="Marketer", max_length=255, required=True)
+
+
+class SavedSearchForm(forms.Form):
+    name = forms.CharField(label="Search Name", max_length=255, required=True)
+    url = forms.CharField(label="Url", max_length=255, required=True, widget=forms.HiddenInput())
```

Comment:
Is 255 characters enough if there's a long URL?

Reply:
This example with a short query "brain bleeding" and the hits per page, section name, section source, and generic name filters is 301 characters: http://localhost:8000/search/es_search?productsection%5Bquery%5D=brain%20bleeding&productsection%5Bmenu%5D%5Bsection_name%5D=Overdose&productsection%5Bmenu%5D%5Bdrug_label_generic_name%5D=Celecoxib&productsection%5BrefinementList%5D%5Bdrug_label_source%5D%5B0%5D=TGA&productsection%5BhitsPerPage%5D=20. I think we should make it ~400?